webapp/app/assessment/fra/sections/originalDataPoint/actions/actions.js

```diff
@@ -88,14 +88,14 @@ export const saveDraft = (countryIso, odp) => (dispatch, getState) => {
   dispatch(persistDraft(countryIso, odp))
 }
 
-export const cancelDraft = (countryIso, odpId, destination) => async () => {
+export const cancelDraft = (countryIso, odpId, destination, history) => async () => {
   if (odpId) {
     // TODO on issue: https://github.com/openforis/fra-platform/issues/154
     // when canceling draft, delete request should respond with odp and then update tables with odp state
     await axios.delete(`/api/odp/draft/?odpId=${odpId}&countryIso=${countryIso}`)
```

Comment:
when canceling draft, delete request should respond with odp and then update tables with odp state

Reply:
This behavior should also be added to delete odp, not only to cancelDraft